src/c++/perf_analyzer/docs/cli.md

```diff
@@ -0,0 +1,710 @@
+<!--
+Copyright 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions
+are met:
+ * Redistributions of source code must retain the above copyright
+   notice, this list of conditions and the following disclaimer.
+ * Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in the
+   documentation and/or other materials provided with the distribution.
+ * Neither the name of NVIDIA CORPORATION nor the names of its
+   contributors may be used to endorse or promote products derived
+   from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Perf Analyzer CLI
+
+## General Options
+
+### `-?`
+### `-h`
+### `--help`
+
+Prints a description of the Perf Analyzer command line interface.
+
+<br>
+
+### `-m`
+
+Specifies the model name for Perf Analyzer to run.
+
+This is a required option.
+
+<br>
+
+### `-x`
+
+Specifies the version of the model to be used. If not specified the most
+recent version (the highest numbered version) of the model will be used.
+
+<br>
+
+### `--service-kind=[triton|triton_c_api|tfserving|torchserve]`
+
+Specifies the kind of service for Perf Analyzer to generate load for. Note: in
+order to use `torchserve` backend, the `--input-data` option must point to a
+JSON file holding data in the following format:
+
+```JSON
+{
+  "data": [
+    {
+      "TORCHSERVE_INPUT": [
+        "<complete path to the content file>"
+      ]
+    },
+    {...},
```

Comment:
It is showing this in red, does it need escaping special characters?

Reply:
This is because it's invalid JSON syntax. I removed the code language suggestion from the code block, so we get rid of the red background. Unfortunately, it doesn't color the different elements of the code differently so we lose some readability, but whatever.